storage/innobase/handler/ha_innodb.cc

```diff
@@ -20233,6 +20233,13 @@ static MYSQL_SYSVAR_ULONGLONG(
     "entire file at once before closing it.",
     NULL, NULL, 0, 0, ~0ULL, UNIV_PAGE_SIZE);
 
+static MYSQL_SYSVAR_ULONGLONG(
+    txlog_init_rate, os_txlog_init_rate, PLUGIN_VAR_OPCMDARG,
+    "The value of this variable determines how fast (in bytes/s) InnoDB "
+    "initializes the transaction log when creating a new file"
+    "Setting this value to 0 means no limit. Default is 128MB",
+    NULL, NULL, 1ULL << 27, 0ULL, ULONG_MAX, 1ULL << 20);
```

Comment:
s/NULL/nullptr/g?

Reply:
Ok, will do this and it is the odd one out :)